Makefile

```diff
@@ -40,7 +40,7 @@ generate: $(SOURCES)
 
 .PHONY: generate-deps
 generate-deps:
-	go get github.com/tools/godep
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
In order for this script to install the correct version (not the latest) I believe you need to set the `DEP_RELEASE_TAG`. That's what I had to do when I installed dep

Reply:
done (with upgrade to 0.4.1 which is the latest anyway)

Reply:
I'd recommend:

```
DEP_RELEASE_TAG=$DEP_RELEASE_TAG curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

That way the env var is only defined for one command.

Reply:
Also, `$$` in needs to be `$`

Reply:
Ah, thanks! fixed